sdk/core/azure-core-amqp/src/amqp/management.cpp

```diff
@@ -269,9 +271,8 @@ namespace Azure { namespace Core { namespace Amqp { namespace _detail {
           case _internal::MessageSenderState::Idle:
           case _internal::MessageSenderState::Closing:
           case _internal::MessageSenderState::Error:
-            Log::Stream(Logger::Level::Error)
-                << "Message Sender Changed State to " << static_cast<int>(newState)
-                << " while management client is open";
+            Log::Stream(Logger::Level::Error) << "Message Sender Changed State to " << newState
```

Comment:
The rule for AMQP is that protocol errors are reported back to the calling component to deal with as it seems appropriate (for instance, eventhubs has retry semantics).

The log levels you pointed to don't have a "it failed but it might be recoverable" category. The closest is informational, which appears to be reserved for "something happened".